manifests/r/RejminetGroupInc/PDFShow/1.1.2/RejminetGroupInc.PDFShow.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with komac v2.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: RejminetGroupInc.PDFShow
+PackageVersion: 1.1.2
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+ProductCode: '{84420C1F-C236-31C3-95DF-80EC13960E1F}'
```

```suggestion
ProductCode: '{4BADBBC9-73F1-3D8C-A247-7777192F2A48}'
```
This also needs to be applied.